reference/conanfile/tools/cmake/cmake.rst

```diff
@@ -47,11 +48,16 @@ constructor
 
 .. code:: python
 
-    def __init__(self, conanfile, build_folder=None):
+    def __init__(self, conanfile, build_folder=None, namespace=None):
```

Comment:
This is outdated, current one is ``def __init__(self, conanfile, parallel=True, namespace=None):``, needs to be fixed (not in this PR)

Reply:
I'll do it

Reply:
https://github.com/conan-io/docs/pull/2250